docs/tutorials/developer-journey/level-1/1.4-using-cycles.mdx

```diff
@@ -91,139 +89,68 @@ The principal will resemble the following format:
 tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe
 ```
 
+## Acquiring cycles
 
-## Acquiring cycles using a cycles coupon
+:::danger
+Cycles are required to deploy to the mainnet. If you only plan to deploy to your local environment for testing, cycles are not required. The following tutorials will focus on local deployment to assure anyone can complete this tutorial series.
```

Comment:
I would also state that devs can move to the next section directly if they do not aim to deploy on mainnet directly.